packages/contracts-bedrock/test/CommonTest.t.sol

```diff
@@ -31,46 +37,186 @@ import { AddressManager } from "src/legacy/AddressManager.sol";
 import { L1ChugSplashProxy } from "src/legacy/L1ChugSplashProxy.sol";
 import { IL1ChugSplashDeployer } from "src/legacy/L1ChugSplashProxy.sol";
 import { CrossDomainMessenger } from "src/universal/CrossDomainMessenger.sol";
+import { GovernanceToken } from "src/governance/GovernanceToken.sol";
 import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
 import { LegacyMintableERC20 } from "src/legacy/LegacyMintableERC20.sol";
+import { LegacyMessagePasser } from "src/legacy/LegacyMessagePasser.sol";
 import { SystemConfig } from "src/L1/SystemConfig.sol";
 import { ResourceMetering } from "src/L1/ResourceMetering.sol";
 import { Constants } from "src/libraries/Constants.sol";
+import { Deploy } from "scripts/Deploy.s.sol";
 
-contract CommonTest is Test {
+contract CommonTest is Deploy, Test {
     address alice = address(128);
     address bob = address(256);
-    address multisig = address(512);
 
-    address immutable ZERO_ADDRESS = address(0);
-    address immutable NON_ZERO_ADDRESS = address(1);
-    uint256 immutable NON_ZERO_VALUE = 100;
-    uint256 immutable ZERO_VALUE = 0;
-    uint64 immutable NON_ZERO_GASLIMIT = 50000;
-    bytes32 nonZeroHash = keccak256(abi.encode("NON_ZERO"));
-    bytes NON_ZERO_DATA = hex"0000111122223333444455556666777788889999aaaabbbbccccddddeeeeffff0000";
+    bytes32 constant nonZeroHash = keccak256(abi.encode("NON_ZERO"));
 
     event TransactionDeposited(address indexed from, address indexed to, uint256 indexed version, bytes opaqueData);
 
     /// @dev OpenZeppelin Ownable.sol transferOwnership event
     event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
 
+    OptimismPortal optimismPortal;
+    L2OutputOracle l2OutputOracle;
+    SystemConfig systemConfig;
+    L1StandardBridge l1StandardBridge;
+    L1CrossDomainMessenger l1CrossDomainMessenger;
+    AddressManager addressManager;
+    L1ERC721Bridge l1ERC721Bridge;
+    OptimismMintableERC20Factory l1OptimismMintableERC20Factory;
+    ProtocolVersions protocolVersions;
+
```

Comment:
Nit, a comment would be nice above each of these blocks to clarify the separation